common/am/lib/src/main/java/org/apache/syncope/common/lib/types/AMEntitlement.java

```diff
@@ -60,6 +60,18 @@
 
     public static final String SAML2_IDP_METADATA_READ = "SAML2_IDP_METADATA_READ";
 
+    public static final String SAML2_SP_METADATA_CREATE = "SAML2_SP_METADATA_CREATE";
```

Comment:
So SAML metadata and keystore cannot be deleted?

Reply:
No, it should not, or it would be catastrophic. Event updates have to be very very carefully planned.